Library/Homebrew/shims/super/mig

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+
 pwd="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
Like in the other comment, I tend to agree. (But as before, not changing functionality and focusing on style changes only was intentional.)